_layouts/fluo-doc.html

```diff
@@ -4,7 +4,7 @@
 <div class="fluo-doc">
 
   {% if page.version != site.latest_fluo_release %}
```

Comment:
@mikewalch  should the check here change?

Reply:
That if statement can be removed as all documentation using this layout is now old.

Reply:
ok I removed it, are the updates ok?

Reply:
yes